OrangeHRMTests/PageObjects/BasePage.cs

```diff
@@ -0,0 +1,58 @@
+﻿using OpenQA.Selenium;
+using OrangeHRMTests.Common.WebElements;
+using OrangeHRMTests.PageObjects.Modules;
+
+namespace OrangeHRMTests.PageObjects
+{
+    public class BasePage
+    {
+        public LeftMenuNavigationPanel LeftMenuNavigationPanel => new LeftMenuNavigationPanel();
+
+        private static MyWebElement AddButton = new MyWebElement(By.XPath("//*[contains(@class, 'orangehrm-header-container')]//button"));
+        private static MyWebElement SaveButton = new MyWebElement(By.XPath("//button[@type='submit']"));
```

Comment:
really not good idea to write locator based on submit property, not reliable at all. your locator should be a unique identifier of element. 
please rewrite to better option, use contains text() ='Save' maybe
and review please other locators, maybe the same problem exists for other elements across all pages